components/carousel/__docs__/site-stories.js

```diff
@@ -0,0 +1,10 @@
+// This object is imported into the documentation site. An example for the documentation site should be part of the pull request for the component. The object key is the kabob case of the "URL folder". In the case of `http://localhost:8080/components/app-launcher/`, `app-launcher` is the `key`. The folder name is created by `components.component` value in `package.json`. The following uses webpack's raw-loader plugin to get "text files" that will be eval()'d by CodeMirror within the documentation site on page load.
+
+/* eslint-env node */
+/* eslint-disable global-require */
+
+const siteStories = [
+	require('raw-loader!@salesforce/design-system-react/components/carousel/__examples__/default.jsx'),
```

Comment:
Unfortunately, the site example only has access to imports from `components/index.js` that means that you can't import the items file `carousel-items.js`. You can import that file in all the other storybook files though.

Reply:
@interactivellama need clarification on this one

Reply:
The Codemirror-using demos on the doc site only have access to exports from `components/index.js`.

`carousel-items.js` is a file that gets imported into doc site stories. This will be undefined on the site. The items should be brought inline into the stories, so folks can copy and paste them out.